README.md

```diff
@@ -479,17 +496,7 @@ The following is a list of enviroment variables supported by Sombra for its conf
-| DD_TRACE_DEBUG                         | no                                                                             | false                                                                                                                      | no     | Enable debug logging in the tracer. Only applicable when you are using Datadog agent for log and metrics collection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| DD_SERVICE_NAME                        | no                                                                             | `transcend-hosted-sombra`                                                                                                  | no     | The name for your Sombra. Only applicable when you are using Datadog agent for log and metrics collection.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
`DD_SERVICE_NAME` is still required though, and it should be set by the customer. It's very useful when forwarding logs, for example